db/migrate/20190304094026_add_identifier_to_policies.rb

```diff
@@ -0,0 +1,6 @@
+class AddIdentifierToPolicies < ActiveRecord::Migration
+  def change
+    add_column :policies, :identifier, :string # not adding null: false here so we can migrate in two steps
```

Comment:
We may need a task to migrate existing policies.

Reply:
For now there is none as it is under rolling updates